inc/customizer/controls/react/typography.php

```diff
@@ -43,7 +43,7 @@ class Typography extends \WP_Customize_Control {
 	 */
 	public function json() {
 		$json                        = parent::json();
-		$json['input_attrs']         = is_array( $this->input_attrs ) ? wp_json_encode( $this->input_attrs ) : $this->input_attrs;
+		$json['input_attrs']         = $this->input_attrs;
 		$json['refresh_on_reset']    = $this->refresh_on_reset;
 		$json['font_family_control'] = $this->font_family_control;
 		return $json;
```

Comment:
I think we should keep the here (input_attrs value)-L46 as json-encoded since our primitive type is array

otherwise that causes some problems
![Screen Shot 2023-03-02 at 22 54 49](https://user-images.githubusercontent.com/25361626/222538062-03e03b44-ea24-4794-8f23-420c789e9d1f.png)

what do you think?

Reply:
Sorry about that, my mistake when dropping the condition. Should be fixed now.

Reply:
No worries, thanks for the fix @preda-bogdan  🚀